RealmSwift/SwiftUI.swift

```diff
@@ -643,6 +653,390 @@ extension EnvironmentValues {
             self[RealmEnvironmentKey] = newValue.configuration
         }
     }
+    /// The current `PartitionValue` that the view should use.
+    public var partitionValue: PartitionValue? {
+        get {
+            return self[PartitionValueEnvironmentKey]
+        }
+        set {
+            self[PartitionValueEnvironmentKey] = newValue
+        }
+    }
+}
+
+@available(iOS 13.0, macOS 10.15, tvOS 13.0, watchOS 6.0, *)
+private class ObservableAsyncOpenStorage: ObservableObject {
+    var app: App?
+    var configuration: Realm.Configuration?
+    var partitionValue: AnyBSON?
```

Comment:
None of these need to be nullable– you can initialise the storage in the propertyWrapper initialiser instead of these being nullable. It also appears that `app` isn't used.